### DISCUSSION TITLE
PR is not accessible and i am facing the same issue

### DISCUSSION THREAD

**Discussion by rajpootAbubakkar** [nuxt/nuxt#10748] [Questions (Nuxt 2)]
There are a couple interweaving issues here.

* If you use `target: 'server'` then asyncData will be fetched on client-side on client-side navigation. (That was your initial issue.)

* If you use `target: 'static'` then asyncData will be fetched at generate-time for all generated pages and no fresh request will be made on client-side navigation. (See [this PR](https://github.com/RomeroGaliza/issues-9474/pull/1) to make this work on your repo.) (If you don't generate the pages, this won't work - that was the second issue which is now fixed by you using router links.)

* However, either way, if you are generating the pages, then there is no server to run asyncData on a per-request basis. If you want that then you should be using `target: 'server'` and running `npm run build && npm run start` instead. (But note that in this case asyncData will _still_ be fetched on client-side on client-side navigation.)

_Originally posted by @danielroe in https://github.com/nuxt/nuxt.js/issues/9474#issuecomment-872863732_

**Comment by danielroe** [comment]
Would you explain more about the issue you are facing?

**Reply by rajpootAbubakkar** [reply]
@danielroe I have real estate site and when i apply search and render from home page to listing page it does not call api as my site target is static and SSR true. so how can i call api as every time search can be different. And to use target static is mandatory. Please help Thank you

**Reply by rajpootAbubakkar** [reply]
@danielroe if you want to take a look at the issue in visual i can share a video with you too for better understanding looking forward to hearing from you. i hope you will help me to solve this issue thank you.
